napari_arboretum/graph.py

```diff
@@ -31,7 +32,7 @@ def is_leaf(self):
         return not self.children
 
 
-def build_reverse_graph(graph: dict) -> list:
+def build_reverse_graph(graph: dict) -> tuple[Union[list, set], dict]:
```

Comment:
technically the `dict` could be expanded here but I don't know what the keys and values are at the moment

Reply:
this is probably `Dict[int, List[int]`